sycl/plugins/hip/CMakeLists.txt

```diff
@@ -44,6 +44,14 @@ else()
   set(PI_HIP_LIB_DIR "${SYCL_BUILD_PI_HIP_LIB_DIR}")
 endif()
 
+# Set up defaults for UR
+set(UR_HIP_INCLUDE_DIR ${PI_HIP_INCLUDE_DIR} CACHE PATH
```

Comment:
I can't remember the rules, but do these expansions need quoting in case of spaces in e.g. `PI_HIP_INCLUDE_DIR`?

Reply:
Might be prudent to add quotes since these might contain paths with spaces.

Reply:
Ah yes, the joys of cmake, done now. Thanks both!